ipam/pool.go

```diff
@@ -214,9 +219,13 @@ func (as *addressSpace) merge(newas *addressSpace) {
 		if pv.epoch < as.epoch {
 			// This pool may have stale addresses.
 			for ak, av := range pv.Addresses {
-				if av.epoch == as.epoch || av.InUse {
+				if av.epoch == as.epoch {
 					// Pool has at least one valid or in-use address.
 					pv.epoch = as.epoch
+				} else if av.InUse {
+					// Mark this address as unhealthy
```

Comment:
A comment that explains why we are doing this would be more helpful: "Address is no longer valid, but still in use."?